executor_manager/executors/docker/executor.py

```
@@ -727,14 +760,21 @@ def _check_container_health(
                         )
-                    except Exception:
-                        pass
+                    else:
+                        try:
+                            self.subprocess.run(
+                                ["docker", "rm", "-f", executor_name],
+                                capture_output=True,
+                                timeout=10,
+                            )
+                        except Exception:
+                            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)